src/mod/databases/mod_pgsql/mod_pgsql.c

```diff
@@ -774,6 +774,11 @@ switch_status_t pgsql_next_result_timed(switch_pgsql_handle_t *handle, switch_pg
 		case PGRES_COPY_IN:
 		case PGRES_COMMAND_OK:
 			break;
+//#if POSTGRESQL_MAJOR_VERSION >= 14
```

Comment:
not going to build on older versions

Reply:
Do you know how to put this condition out of the switch since gcc-11.2.x does not allow it.

Reply:
ok, I fixed the conflicts with master, but anyhow it cannot compile with gcc-11.x.x for now

Reply:
ok my bad, it seems that now the issue went away. consider this patch ok.
however the other issue I posted is still persisting (#1440 )

Reply:
I sent another PR that fixes the problem on the #if #endif POSTGRESQL_MAJOR condition into the switch.
gcc 11.2 does not allow a condition into a switch unless the "default: break;" is specified.
everything is compiling fine now (with all the patches I sent)